.github/workflows/_parse_version.yml

```diff
@@ -84,6 +84,12 @@ jobs:
           ref: ${{ inputs.commit_sha }}
         timeout-minutes: 5
 
+      - name: Install python
+        uses: actions/setup-python@29a37be0a3d3e8bf5bc1eb19cd0502922f5b312a  # pin v5.2.0
+        id: setup-python
+        with:
+          python-version: 3.12
+
```

Comment:
Why do you need to install python in `Parse version` job ? the version provided by the system is not good ?

Reply:
We want the same version of python to be used everywhere to avoid unexpected issues.

Typically, `tomllib` has been introduced in python 3.11